src/main/java/org/terasology/tutorialWorldGeneration/HouseRasterizer.java

```diff
@@ -40,20 +41,20 @@ public void initialize() {
     public void generateChunk(CoreChunk chunk, Region chunkRegion) {
         HouseFacet houseFacet = chunkRegion.getFacet(HouseFacet.class);
 
-        for (Entry<BaseVector3i, House> entry : houseFacet.getWorldEntries().entrySet()) {
+        for (Entry<Vector3ic, House> entry : houseFacet.getWorldEntries().entrySet()) {
             // there should be a house here
             // create a couple 3d regions to help iterate through the cube shape, inside and out
             Vector3i centerHousePosition = new Vector3i(entry.getKey());
             int extent = entry.getValue().getExtent();
             centerHousePosition.add(0, extent, 0);
-            Region3i walls = Region3i.createFromCenterExtents(centerHousePosition, extent);
-            Region3i inside = Region3i.createFromCenterExtents(centerHousePosition, extent - 1);
+            BlockRegion walls = BlockRegion.createFromCenterExtents(centerHousePosition, extent);
+            BlockRegion inside = BlockRegion.createFromCenterExtents(centerHousePosition, extent - 1);
 
             // loop through each of the positions in the cube, ignoring the is
-            for (Vector3i newBlockPosition : walls) {
-                if (chunkRegion.getRegion().encompasses(newBlockPosition)
-                        && !inside.encompasses(newBlockPosition)) {
-                    chunk.setBlock(ChunkMath.calcRelativeBlockPos(newBlockPosition), stone);
+            for (Vector3ic newBlockPosition : walls) {
+                if (chunkRegion.getRegion().contains(newBlockPosition)
+                        && !inside.contains(newBlockPosition)) {
+                    chunk.setBlock(Chunks.toRelative(newBlockPosition), stone);
```

Comment:
Since we need to provide Vector3ic,

```suggestion
                    chunk.setBlock(Chunks.toRelative(new Vector3i(newBlockPosition)), stone);
```